docs/advanced/admin-commands.md

```diff
@@ -1,58 +1,245 @@
+- `ethnicities_with_issues`: Displays a list of ethnicities and "True" or
+  "False" indicating if that ethnicity is represented in the collection.
+- `load_batches`: Runs the `load_batch` command for each batch in a given file.
+- `make_countries_fixture`: Loads MARC Country list XML from the web, and dumps
+  JSON fixture to stdout.  *Seems to be broken.*
+- `process_coordinates`: Rebuilds a batch's coordinates file; unless a batch
+  ingest goes horribly awry *and* you can't reingest said batch, this shouldn't
+  need to be run.
+- `update_has_issues`: Fixes titles that are not reporting issues.  This
+  shouldn't need to be run unless data is being inserted into the database
+  manually.
+- `update_sitemap`: Creates a sitemap for search engines, but currently this
+  functionality isn't complete.  We will potentially improve this command in
+  the future, but for now it has to be considered unsupported.
+
+### Supported
+
+These commands will see continued maintenance and support in Open ONI for the
+foreseeable future.
+
+- [Indexing commands](#indexing) (`index`, `index_pages`, `index_titles`,
+  `setup_index`, and `zap_index`)
+- [`load_batch`](#load_batch)
+- [`load_copyright`](#load_copyright)
+- [`load_copyright_map`](#load_copyright_map)
+- [`load_titles`](#load_titles)
+- [`purge_batch`](#purge_batch)
+- [`purge_django_cache`](#purge_django_cache)
+
+## `diff_batches`
+
+Given a file with batch names, shows a list of batches and an indicator of
+their status relative to the current system's batch list.  If a batch is in the
+system but not in the file, it is prefixed with a plus `+`.  If a batch is in
+the system and the file, it is prefixed with a space ` `.  If a batch is not in
+the system but is listed in the file, it is prefixed with a minus `-`.
+
+e.g., given the following batches loaded:
+
+```
+batch_dlc_manyyears_ver01
+batch_mnhi_german_ver01
+batch_nbu_manyissues_ver01
+```
+
+And a file called `batch_list` with the following:
+
+```
+batch_dlc_manyyears_ver01
+batch_nbu_manyissues_ver01
+batch_oru_foo_ver01
+```
+
+The command `diff_batches batch_list` would print this:
+
+```
+ batch_dlc_manyyears_ver01
++batch_mnhi_german_ver01
+ batch_nbu_manyissues_ver01
+-batch_oru_foo_ver01
+```
+
+## Indexing
+
+There are five commands used to manage the Solr index.  Roughly in order of
+need, they are as follows.
+
+### Setup
+
+`setup_index` **must** be run when the application is first being set up.  It
+tells Solr about all the fields we use, and configures Solr in a way that works
+for Open ONI's needs.  If this is not run, ONI *will not function*.
+
+### Updating
+
+`index` rebuilds the entire title and page index data in Solr, including the
+page OCR data.  It shouldn't be necessary most of the time, but it can be
+useful to run if Solr data becomes corrupt (though this is a very rare
+occurrence), or in cases the Solr index must be deleted, e.g., if you upgrade
```

Comment:
I think I would prefer language like "in cases the Solr index must be rebuilt," since if you were upgrading you are not actually required to delete your old index, but you do need a new index

Reply:
If you upgrade Solr by one major version, they recommend destroying and rebuilding the index.  If you upgrade by two major versions *since the start of your index's life*, you "must" destroy and reindex.

I actually feel like we may want to call out this fact more than just in the advanced docs, though I'm not sure where.

Reply:
Ahhh okay.